frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -41,3 +45,11 @@ export const getTemplateValidationsFromTemplate = (
     return new TemplateValidations();
   }
 };
+
+export const getTemplateMemory = (template: TemplateKind): string => {
+  const baseMemoryValue = convertToBaseValue(getMemory(selectVM(template)));
+  return humanizeBinaryBytes(baseMemoryValue).string;
+};
+
+export const getTemplateFlavorDesc = (template: TemplateKind): string =>
```

Comment:
there is a function called getFlavorText. Can you please modify it so the flavor would be optional there?. 
Also it is not a good idea to add such specific functions like getTemplateMemory to the template since template is a generic concept and this is a domain of a VM. Eg it is easier to create a VMWrapper and all the functions are there